Tabloid/Controllers/CommentsController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc;
+using Tabloid.Repositories;
+using Tabloid.Models;
+
+namespace Tabloid.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
```

Comment:
Need to add [Authorize] to class definition to ensure that users are authenticated and authorized

Reply:
Added!